### PR TITLE
enable logs feature in hack/local-up-kubeedge script

### PR DESCRIPTION
**Pull request by YaozhongZhang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind feature

**What this PR does / why we need it**:
enable `logs` feature in `hack/local-up-kubeedge` script

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: hack/local-up-kubeedge.sh [hunk 1]
```diff
@@ -90,6 +93,7 @@ function start_cloudcore {
   CLOUD_CONFIGFILE=${KUBEEDGE_ROOT}/_output/local/bin/cloudcore.yaml
   CLOUD_BIN=${KUBEEDGE_ROOT}/_output/local/bin/cloudcore
   ${CLOUD_BIN} --minconfig >  ${CLOUD_CONFIGFILE}
+  sed -i '/modules:/a\  cloudStream:\n    enable: true\n    streamPort: 10003\n    tlsStreamCAFile: /etc/kubeedge/ca/streamCA.crt\n    tlsStreamCertFile: /etc/kubeedge/certs/stream.crt\n    tlsStreamPrivateKeyFile: /etc/kubeedge/certs/stream.key\n    tlsTunnelCAFile: /etc/kubeedge/ca/rootCA.crt\n    tlsTunnelCertFile: /etc/kubeedge/certs/server.crt\n    tlsTunnelPrivateKeyFile: /etc/kubeedge/certs/server.key\n    tunnelPort: 10004' ${CLOUD_CONFIGFILE}
```
Comment (daixiang0) [inline hack/local-up-kubeedge.sh:96]: Can be simpified, only replace what we need.

File: hack/local-up-kubeedge.sh [hunk 2]
```diff
@@ -167,12 +204,16 @@ kubectl create ns kubeedge
...
 
+generate_streamserver_cert
+
 start_cloudcore
 
 sleep 2
 
 start_edgecore
 
+sudo iptables -t nat -A PREROUTING -p tcp --dport 10350 -j REDIRECT --to-port 10003
```
Comment (daixiang0) [inline hack/local-up-kubeedge.sh:215]: Put this in function and check exit code.

File: hack/local-up-kubeedge.sh (hunk 1 above)
Comment (YaozhongZhang) [reply hack/local-up-kubeedge.sh:96]: @daixiang0 `cloudcore --minconfig` does not print the `cloudStream` section

File: hack/local-up-kubeedge.sh (hunk 2 above)
Comment (YaozhongZhang) [reply hack/local-up-kubeedge.sh]: done

File: hack/local-up-kubeedge.sh [hunk 3]
```diff
@@ -167,6 +206,8 @@ kubectl create ns kubeedge
 create_device_crd
 create_objectsync_crd
 
+generate_streamserver_cert
```
Comment (fisherxu) [inline hack/local-up-kubeedge.sh:209]: Can the streamserver use certs in cloudhub? @YaozhongZhang @ls889

**Comment by ls889** [comment]
@fisherxu Now streamserver certificate is manually generated.  it's mainly to communicate with apiserver. so steamserver can't reuse Cloudhub’s certificate.

**Comment by daixiang0** [comment]
So is there any chance to reuse?

File: hack/local-up-kubeedge.sh (hunk 1 above)
Comment (daixiang0) [reply hack/local-up-kubeedge.sh:96]: I mean that you write too mush in `sed` replace.

Comment (YaozhongZhang) [reply hack/local-up-kubeedge.sh:96]: I know that. But there was only `cloudHub` moudle in `cloudcore --minconfig`, I have to append all these lines of `cloudStream` moudle configuration.

Comment (daixiang0) [reply hack/local-up-kubeedge.sh:96]: I see.

**Pull request merged**